shared/common-adapters/plain-input.d.ts

```diff
@@ -1,23 +1,21 @@
-// @flow
 import * as React from 'react'
-import {type StylesCrossPlatform} from '../styles'
-import type {TextType} from './text'
+import {StylesCrossPlatform} from '../styles'
+// @ts-ignore not converted
+import {TextType} from './text'
 
 export type KeyboardType =
   | 'default'
   | 'email-address'
-  | 'numeric'
+  | 'numeric' // iOS only
```

Comment:
These comments are in the wrong place. (Sorry this is hard for the parser to pick up properly)